homework9/hw9task3/tests/test_universal_file_counter.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from iterators_3.universal_file_counter import universal_file_counter
+
+path = os.path.dirname(__file__)
+
+
+def test_universal_file_counter_with_tokenizer_data_split_by_space():
+    assert universal_file_counter(path, "txt", str.split) == 11
+    assert universal_file_counter(path, "txt", lambda data: data.split()) == 11
```

Comment:
This lambda is also an example of unnecessary wrapping.  You wrap callable into callable with the same signature.

Reply:
Agree, but how without lambda to check how many tokens we have if tokenizer=`data.split('some symbol')` ? Or we don`t need to check this in this task? Yes, in the task we are given tokenizer as str.split.

Reply:
As I understand you correctly you are asking me if `lambda data: data.split()` and `lambda data: data.split("c")` are doing the same?  

No, these function has completely different behavior.  The first one could be simplified and the second one no.